shell/platform/darwin/ios/ios_context_gl.mm

```diff
@@ -24,7 +25,11 @@
   }
 }
 
-IOSContextGL::~IOSContextGL() = default;
+IOSContextGL::~IOSContextGL() {
+  if (main_context_) {
+    main_context_->releaseResourcesAndAbandonContext();
```

Comment:
I don't know if it's possible. The various GPUSurfaceGL still has this shared pointer right? Could there be any risks?